python_modules/dagster/dagster/_core/definitions/asset_check_spec.py

```diff
@@ -55,7 +56,7 @@ class AssetCheckSpec(
         name (str): Name of the check.
         asset_key (AssetKey): The key of the asset that the check applies to.
         description (Optional[str]): Description for the check.
-        severity (AssetCheckSeverity): Severity of the check.
+        severity (AssetCheckSeverity): Severity of the check. Defaults to WARN
```

Comment:
Let's reference `WARN` using the actual pydoc string so that it's linked in the rendered API docs.

Reply:
do you have an example of that

Reply:
None of :data, :class,  :meth seem to work. I think it's alright because there the severity enum is linked on the same line

<img width="589" alt="Screenshot 2023-08-29 at 9 38 27 PM" src="https://github.com/dagster-io/dagster/assets/22018973/325f64e0-c8c6-4343-bcea-4dd73db22bd4">